scripts/machine-launch-script.sh

```diff
@@ -0,0 +1,305 @@
+#!/bin/bash
+
+CONDA_INSTALL_PREFIX=/opt/conda
+CONDA_INSTALLER_VERSION=4.12.0-0
+CONDA_INSTALLER="https://github.com/conda-forge/miniforge/releases/download/${CONDA_INSTALLER_VERSION}/Miniforge3-${CONDA_INSTALLER_VERSION}-Linux-x86_64.sh"
+CONDA_CMD="conda" # some installers install mamba or micromamba
+CONDA_ENV_NAME="chipyard"
+
+DRY_RUN_OPTION=""
+DRY_RUN_ECHO=()
+REINSTALL_CONDA=0
+
+usage()
+{
+    echo "Usage: $0 [options]"
+    echo
+    echo "Options:"
+    echo "[--help]                  List this help"
+    echo "[--prefix <prefix>]       Install prefix for conda. Defaults to /opt/conda."
+    echo "                          If <prefix>/bin/conda already exists, it will be used and install is skipped."
+    echo "[--env <name>]            Name of environment to create for conda. Defaults to '$CONDA_ENV_NAME'."
+    echo "[--dry-run]               Pass-through to all conda commands and only print other commands."
+    echo "                          NOTE: --dry-run will still install conda to --prefix"
+    echo "[--reinstall-conda]       Repairs a broken base environment by reinstalling."
```

Comment:
When is this `--reinstall-conda` thing useful?

Reply:
If for example, you have removed python from your base environment or otherwise broken conda.  It's a passthrough to the conda installer.